client/package.json

```diff
@@ -1,33 +1,32 @@
 {
   "name": "client",
-  "version": "0.1.0",
-  "private": true,
+  "proxy": "http://localhost:5000",
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
-    "@fontsource/poppins": "^4.5.5",
+    "@fontsource/poppins": "^4.5.8",
     "@mui/icons-material": "^5.6.2",
-    "@mui/material": "^5.6.2",
-    "axios": "^0.26.1",
+    "@mui/material": "^5.6.4",
+    "@mui/styled-engine-sc": "^5.6.1",
+    "@types/jest": "^27.5.0",
```

Comment:
shouldn't this be listed in devDependencies? 
in most cases anything that relates to typescript should be listed in devDependencies.

this article should help you decide:
https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies